packages/firestore/src/api/bundle.ts

```diff
@@ -138,3 +116,30 @@ export class LoadBundleTask
     }
   }
 }
+
+export function loadBundle(
+  db: Firestore,
+  bundleData: ArrayBuffer | ReadableStream<Uint8Array> | string
+): LoadBundleTask {
+  const resultTask = new LoadBundleTask();
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  firestoreClientLoadBundle(
```

Comment:
You should drop the Promise from `firestoreClientLoadBundle`. It doesn't seem to be needed.